meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -44,14 +48,25 @@ OSTRO_VM_IMAGE_TYPES = "dsk.xz dsk.zip dsk.vdi.zip"
 # which must contain only alphanumeric symbols,'-' and '_'.
 # Any other symbols would be skipped in parser.
 #
-# Following targets would be used to perform default build task:
-OSTROPROJECT_CI_BUILD_TARGETS="${DISTRO}-image ${DISTRO}-image-dev"
+# Following targets would be used to perform default build task.
+OSTROPROJECT_CI_BUILD_TARGETS="ostro-image ostro-image-qa ostro-image-reference"
 # Following targets would be executed with do_populate_sdk task
 OSTROPROJECT_CI_SDK_TARGETS=""
-# Following targets would be executed with do_populate_sdk_ext task
-OSTROPROJECT_CI_ESDK_TARGETS="${DISTRO}-image"
-# Following targets would be executed with do_test_iot_export task
-OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
+# Following targets would be executed with do_populate_sdk_ext task.
+# Uses the fully-populated image to ensure that the eSDK is as complete
+# as possible.
+OSTROPROJECT_CI_ESDK_TARGETS="ostro-image-all"
+# Following targets would be executed with do_test_iot_export task.
+# Must be "ostro-image", the task seems to expect that name.
+OSTROPROJECT_CI_TEST_EXPORT_TARGETS="ostro-image"
+
+# ostro-image as build target above shall only produce bundles,
+# but no images.
+OSTRO_VM_IMAGE_TYPES_pn-ostro-image = ""
+IMAGE_FSTYPES_pn-ostro-image = ""
```

Comment:
If content for ostro-image recipe is produced during local build of the user and it is usable, it's ok then to produce same content in automated build. Producing images that are different in form of compression is ok to be deviated in automatic and local build. Producing **additional** artifacts in automatic manner (bundles for swupd) which are not always usable by local users it is also ok. But **not producing** something that is content different from what normal developer would see is not ok in my opinion.